src/Bicep.Core/Registry/OciModuleRegistry.cs

```diff
@@ -106,6 +107,14 @@ public override bool TryGetLocalModuleEntryPointUri(OciArtifactModuleReference r
             if (!ociAnnotations.Any() || (ociAnnotations.TryGetValue(LanguageConstants.OciOpenContainerImageDocumentationAnnotation, out string? documentationUri) &&
                 string.IsNullOrWhiteSpace(documentationUri)))
             {
+                if (ociArtifactModuleReference.Registry == "mcr.microsoft.com")
+                {
+                    var tag = ociArtifactModuleReference.Tag;
+                    var repository = Regex.Replace(ociArtifactModuleReference.Repository, "bicep/", string.Empty);
```

Comment:
>Replace [](http://example.com/codeflow?start=43&length=7)

Couldn't this replace any occurrence of "bicep/" in the repository string?

Instead, could you change L110 to also check that repository starts with "bicep/" and then just get the repository by getting a substring of the original repository that starts at `"bicep/".Length`?

Reply:
Would also be good to put both the "mcr.microsoft.com" string and "bicep/" string in constants (local or in this class).

Reply:
Updated here:
https://github.com/Azure/bicep/commit/973ebc7f30cc2cc1610fa81a30ae74b2ec0bd25c

Thanks!

Reply:
This would also make the check more resilient to cases where other Bicep modules are published to MCR that have different doc links.